lib/kramdown_rpf/rpf.rb

```diff
@@ -112,6 +119,29 @@ def self.convert_print_only_to_html(content)
         ::Kramdown::Document.new("<div class=\"u-print-only\">\n#{content}</div>", KRAMDOWN_OPTIONS).to_html
       end
 
+      def self.convert_knowledge_quiz_question_to_html(question, _indent)
+        question_match = QUESTION_REGEXP.match(question)
+        return '' if question_match.nil? || question_match.length() < 2
```

Comment:
Useless empty brackets after `length`
```suggestion
        return '' if question_match.nil? || question_match.length < 2
```